feat(api): new Node.js SDK (#2147)
openmeterio/openmeter: api/client/javascript/README.md

# OpenMeter JavaScript SDK

## Install

```sh
npm install --save @openmeter/sdk
```

## Configuration for accessing the OpenMeter API

> To use the OpenMeter SDK on your backend, you need to configure it the `baseUrl` and `apiKey` for OpenMeter Cloud:

Reviewer: ```suggestion
To use the OpenMeter SDK on your backend, you need to configure `baseUrl` and `apiKey` for OpenMeter Cloud:
```

```ts
import { OpenMeter } from '@openmeter/sdk'

const openmeter = new OpenMeter({
  baseUrl: 'https://openmeter.cloud',
  apiKey: 'om_...',
})
```

## Configuration for accessing the OpenMeter Portal API

> To use the OpenMeter Portal SDK on your frontend, you need to configure it the `baseUrl` and `portalToken`:

Reviewer: ```suggestion
To use the OpenMeter Portal SDK on your frontend, you need to use a portal token in your configuration:
```
